crates/router/src/core/payment_methods/cards.rs

```diff
@@ -405,15 +405,21 @@ pub async fn migrate_payment_method(
     merchant_id: &id_type::MerchantId,
     merchant_account: &domain::MerchantAccount,
     key_store: &domain::MerchantKeyStore,
-) -> errors::RouterResponse<api::PaymentMethodResponse> {
+) -> errors::RouterResponse<api::PaymentMethodMigrateResponse> {
     let mut req = req;
-    let card_details = req.card.as_ref().get_required_value("card")?;
+    let card_details = &req.card.clone().get_required_value("card")?;
 
     let card_number_validation_result =
         cards::CardNumber::from_str(card_details.card_number.peek());
 
-    let card_bin_details =
-        populate_bin_details_for_masked_card(card_details, &*state.store).await?;
+    let skip_card_expiry_validation = req.skip_card_expiry_validation.unwrap_or(false); //here if `skip_card_expiry_validation` is not present in the request it will be set to false
```

Comment:
This comment is not needed,
Generally use comments to explain why rather than how or what
you can also use `unwrap_or_default`